acceptance/framework/flags/flags.go

```diff
@@ -55,6 +56,7 @@ func (t *TestFlags) init() {
 	flag.StringVar(&t.flagNamespace, "namespace", "", "The Kubernetes namespace to use for tests.")
 
 	flag.StringVar(&t.flagConsulImage, "consul-image", "", "The Consul image to use for all tests.")
+	flag.IntVar(&t.flagConsulMajorVersion, "consul-major-version", 0, "The Consul major version.")
```

Comment:
Alternatively I could use the `go-version` library to track the whole consul version throughout, but since this is just for acceptance tests so I'm on the fence about it.